README.md

```diff
@@ -119,9 +119,69 @@ Connection: keep-alive
 
 This error shows the request is missing a required property `name` from the HTTP request body.
 
+#### Server Validation
+
+OpenAPI lets API spec authors make only certain servers available, and they also allow certain operations to be restricted to certain servers. Make sure the server URL you plan to use is a valid server this the particular operation you are attempting.
+by providing a `__server` query param.
+
+Take this minimalistic spec (oas3) example:
+
+```yaml
+---
+openapi: 3.0.2
+paths:
+  '/pet':
+    get:
+      responses:
+        '200':
+          content:
+            '*/*':
+              schema:
+                type: string
+servers:
+  - url: '{schema}://{host}/{basePath}'
+    variables:
+      schema:
+        default: http
+        enum:
+          - http
+          - https
+      host:
+        default: stoplight.io
+        enum:
+          - stoplight.io
+          - dev.stoplight.io
+      basePath:
+        default: api
+```
+
+You can make a request enforcing server validation by providing the `__server` query string parameter:
+
+```bash
+curl -X GET localhost:4010/pet?__server=http://stoplight.io/api
+# This will return a 200 response
+```
+
+On the other hand, putting a server which is not defined in the specification, for example:
+
+```bash
+curl -X GET localhost:4010/pet?__server=ftp://acme.com/api
```

Comment:
Oh this is weird that we introduced FTP here just without any explanation, just snuck it in there. I honestly do not know if FTP is even allowed in OpenAPI, it doesn't mentioned either way but... it has never come up. https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#serverObject